app/assets/stylesheets/application-ie8.scss

```diff
@@ -1,6 +1,5 @@
 // BASE STYLESHEET FOR IE 8 COMPILER
 
-$is-ie: true;
-$ie-version: 8;
+$govuk-is-ie8: true;
```

Comment:
I haven't seen this before, is this documented somewhere in the Design System? I thought these were custom variables for this application.

Reply:
with govuk-frontend you can build an ie8 stylesheet - docs here https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/supporting-internet-explorer-8.md

Reply:
Just so I understand, where did the previous `$is-ie` stuff come from? What makes this work?

Incidentally I just checked a few pages in this PR in IE8 and it actually looks better than the current live version 😲 

<img width="1097" alt="Screen Shot 2019-09-05 at 10 08 26" src="https://user-images.githubusercontent.com/861310/64328346-31a64d00-cfc5-11e9-8673-a0e461315b7f.png">

<img width="1109" alt="Screen Shot 2019-09-05 at 10 08 47" src="https://user-images.githubusercontent.com/861310/64328368-39fe8800-cfc5-11e9-8088-9459faec692f.png">

Reply:
is-ie and ie-version come from frontend_toolkit. in govuk-frontend 'ie-version' was replaced with just `$govuk-is-ie8` as we dropped support of 6 and 7